monitoring/monitoring/monitoring.go

```diff
@@ -66,6 +66,20 @@ func (c *Container) validate() error {
 	return nil
 }
 
+// isAlertDefined indicates if a dashboard has any alert defined.
+func (c *Container) isAlertDefined() bool {
```

Comment:
Alerts should be defined as a preferred default, so this condition should be inverted:

```suggestion
func (c *Container) noAlertsDefined() bool {
```